banksystem/accounts/views.py

```diff
@@ -1,22 +1,23 @@
-from django.shortcuts import render
-from django.views.generic import TemplateView
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+from rest_framework.views import APIView
 
 from .utils import get_user_bank_accounts
+from .serializer import AccountSerializer
 
```

Comment:
"/accounts/" return all the accounts for that user

"/accounts/1" return detail for that particular account